cmd/akash/key.go

```diff
@@ -81,3 +81,36 @@ func doKeyListCommand(session session.Session, cmd *cobra.Command, args []string
 	tw.Flush()
 	return nil
 }
+
+func keyShowCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "show [name]",
+		Short: "display a key",
+		RunE:  session.WithSession(session.RequireRootDir(doKeyShowCommand)),
+	}
+	session.AddFlagKeyType(cmd, cmd.Flags())
+	return cmd
+}
+
+func doKeyShowCommand(session session.Session, cmd *cobra.Command, args []string) error {
+	if len(args) != 1 {
```

Comment:
Declare this in the command like [here](https://github.com/ovrclk/akash/blob/41e00ba0cf7ef5c811917ef9a6018686aaa5e280/cmd/akash/provider.go#L50)